src/components/ObsDetails/ObsDetailsContainer.js

```diff
@@ -416,6 +458,44 @@ const ObsDetailsContainer = ( ): Node => {
     createIdentificationMutation.mutate( { identification: idParams } );
   };
 
+  const onSuggestId = useCallback( ( ) => {
+    // check for disagreement
+    let observationTaxon = observation.taxon;
```

Comment:
For code like this, it can be helpful to leave few comment lines above with a link to the existing iNaturalist web code. That way other devs know where it's coming from and it's easy to double check that it's actually at parity with the web code.

```
// based on disagreement code in iNat web
// https://github.com/inaturalist/inaturalist/blob/30a27d0eb79dd17af38292785b0137e6024bbdb7/app/webpack/observations/show/ducks/observation.js#L827-L838
```